docs/use/RestoreBackup.md

```diff
@@ -1,16 +1,28 @@
-
 # Restore the database from a previously saved backup
 
+Database backups and data replication are complementary components of an effective disaster recovery strategy. 
+
+Everest enables you to use backups to either create a new database or restore the backup data to the original database. Choosing between these two options often depends on the specific scenario, the urgency of recovery, the impact on ongoing operations, the need for data isolation, and the purpose of the restoration (such as testing, recovery from corruption, or analysis).
+
+## Restore database from backup
+
+Restoring to the same cluster is useful in the following scenarios:
+
+- **When restoring from a Point-in-time recovery (PITR) timestamp** as PITR does not currently support restoring to a new database.
+- **For faster recovery**: It's typically quicker to restore data to an existing database as it eliminates the need for creating a new database structure.
+- **Minimal disruption**: Restoring to the existing database ensures continuity in operations and minimizes disruptions since it doesn’t involve switching between different database instances.
+- **Database consistency:** Maintains the same database ID, configurations, and dependencies, ensuring consistency within the existing environment.
+- **Historical reference**: Retains historical data and schema within the same database, which might be useful for certain types of analysis or comparisons.
```

Comment:
I'm not sure about this either. In my view, doing these restore operations changes the status of the DB to a previous point in time so I find it hard to argue that this is a pro in comparison with creating a new DB.

Reply:
removed